### PR TITLE
[Bug Fix] Fix for dual wield animation when same delay weapons.

### PR DESCRIPTION
**Pull request by KayenEQ**
Resolves an issue where if you start auto attacking with two weapons with the same delay the offhand animation will never fire due to the throttle timer, the primary would always go first. This solution checks when the timers are set if both attack and dw timers are the same, if so it will ensure an equal chance of primary and secondary animations.

### CODE REVIEW

File: zone/attack.cpp [hunk 1]
```diff
@@ -5477,6 +5489,21 @@ void Client::SetAttackTimer()
...
+			primary_speed = speed;
+		}
+		else if (i == EQ::invslot::slotSecondary) {
+			secondary_speed = speed;
+		}
+	}
+
+	//To allow for duel wield animation to display correctly if both weapons have same delay
+	if (primary_speed == secondary_speed) {
+		SetDualWeildingSameDelayWeapons(1);
```
Comment (Akkadius) [inline zone/attack.cpp:5503]: Weild vs Wield

**Comment by Akkadius** [comment]
I'll look at this more later, my eyes are going cross eyed :sleeping:

File: zone/attack.cpp (hunk 1 above)
Comment (KayenEQ) [reply zone/attack.cpp]: I REFUSE to learn how to spell DUELWEILD

**Review by Natedog2012 — APPROVED** [review]
The visuals for dual wield working much better when both weapons are the same speed

**Review by mackal — APPROVED** [review]

**Comment by mackal** [comment]
I feel like there has to be a better way to do this, but oh well. As long as this improves things it should be fine.

**Pull request merged**